### PR TITLE
Add Canopus (OPUS) to defi.tokenlist.json

### PR DESCRIPTION
**Pull request by canopus-network**

### CODE REVIEW

File: defi.tokenlist.json [hunk 1]
```diff
@@ -14,6 +14,14 @@
     "timestamp": "2021-05-01T22:00:00+00:00",
     "tokens": [
         {
+            "address": "0x76076880e1EBBcE597e6E15c47386cd34de4930F",
+            "chainId": 43114,
+            "name": "Canpous",
+            "symbol": "OPUS",
+            "decimals": 18,
+            "logoURI": "https://raw.githubusercontent.com/ava-labs/bridge-tokens/main/avalanche-tokens/0x76076880e1EBBcE597e6E15c47386cd34de4930F/logo.png
```
Comment (Jonasslv) [inline defi.tokenlist.json:22]: missing double quotes at the end of URI

File: defi.tokenlist.json [hunk 2]
```diff
@@ -14,6 +14,14 @@
     "timestamp": "2021-05-01T22:00:00+00:00",
     "tokens": [
         {
+            "address": "0x76076880e1EBBcE597e6E15c47386cd34de4930F",
+            "chainId": 43114,
+            "name": "Canpous",
```
Comment (Jonasslv) [inline defi.tokenlist.json:19]: wrong token name, isnt "Canopus"?

Comment (canopus-network) [reply defi.tokenlist.json:19]: Sorry, yes its Canopus

**Comment by dasconnor** [comment]
you need to rebase and bump the token version. Also, please add your token entry to the end of the list.

**Pull request closed without merging**